.github/workflows/ci.yml

```diff
@@ -149,7 +149,8 @@ jobs:
             cpanm --test-only --verbose Dancer2::Template::Handlebars
             cpanm --test-only --verbose Dancer2::Session::Cookie
             cpanm --test-only --verbose Dancer2::Plugin::Email
-            cpanm --test-only --verbose Dancer2::Plugin::Auth::Extensible
+            # This has a broken test that needs fixing. Uncomment when resolved!
```

Comment:
AFAICT this is fixed in the Git repo of Dancer2::Plugin::Auth::Extensible: https://github.com/PerlDancer/Dancer2-Plugin-Auth-Extensible/commit/04352bfd06c629329a81e3ec0704783718d0ac0b.

Please contact @abeverley about a release.

Reply:
> Please contact @abeverley about a release

Sorry, I forgot that release was waiting. Will aim to get done today.

Reply:
Thank you! I'll put this back before I merge and release.